contracts/tokenomics/generator/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "astroport-generator"
-version = "1.0.0"
+version = "1.1.1"
```

Comment:
Since this PR is a fix we need to increase only patch version i.e I suggest 1.0.1 version.

Reply:
That's a feature to claim asset rewards from pools. Will set it to 1.1.0